File: GarbageV5/app/src/main/java/dk/itu/garbagev5/MainActivity.java
package dk.itu.garbagev5;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    private static ItemsDB itemsDB;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.garbage);

        itemsDB = ItemsDB.get();
        setUpFragments();
    }

    private void setUpFragments() {
        FragmentManager fm = getSupportFragmentManager();
        Fragment fragmentSearch= fm.findFragmentById(R.id.container_search);
        Fragment fragmentList= fm.findFragmentById(R.id.container_list);
        if ((fragmentSearch == null) && (fragmentList == null)) {
            fragmentSearch= new SearchFragment();
            fragmentList= new ListFragment();
            fm.beginTransaction()
                    .add(R.id.container_search, fragmentSearch)
                    .add(R.id.container_list, fragmentList)
                    .commit();
        }
    }
}

File: GarbageV7/app/src/main/java/dk/itu/garbagev7/MainActivity.kt
package dk.itu.garbagev7

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment

class MainActivity : AppCompatActivity() {
    private var itemsDB: ItemsDB? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.garbage)

        itemsDB = ItemsDB.get()
        setUpFragments()

    }

    private fun setUpFragments() {
        val fm = supportFragmentManager
        var searchFragment:Fragment? = fm.findFragmentById(R.id.container_search)
        var listFragment:Fragment? = fm.findFragmentById(R.id.container_list)
        if ((searchFragment == null) && (listFragment == null)) {
            fm.beginTransaction()
                .add(R.id.container_search, Search())
                .add(R.id.container_list, ListFragment())
                .commit()
        }
    }


}